### PR TITLE
Added flush cache and SEO tuning

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**

### CODE REVIEW

File: docs/develop/api-reference/index.mdx [hunk 1]
```diff
@@ -196,6 +196,22 @@ puts client.caches.to_a.join(", ")
...
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache), simply flushing a cache keeps all settings for this cache and only deletes the data in the cache.
```
Comment (allenheltondev) [inline docs/develop/api-reference/index.mdx]: This sentence uses the word *cache* 5 times. Maybe reword it to something like 
```
While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache), simply flushing a cache keeps all settings and only deletes the data in it.
```

Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: Changed wording and pushed new revision.

File: docs/develop/api-reference/index.mdx [hunk 2]
```diff
@@ -196,6 +196,93 @@ puts client.caches.to_a.join(", ")
...
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
+
+:::
+
+<SdkExamples
+  js={`
+const authToken="eyJhbGc.MyTestToken";
+const defaultTTL = 15;
+const momento = new SimpleCacheClient(authToken, defaultTtl);
```
Comment (allenheltondev) [inline docs/develop/api-reference/index.mdx]: These aren't the arguments for the constructor anymore. 

*Taken from the [npmjs page for the SDK](https://www.npmjs.com/package/@gomomento/sdk)*

```javascript
const credentialsProvider = new EnvMomentoTokenProvider('MOMENTO_AUTH_TOKEN');

const loggerOptions: LoggerOptions = {
  level: LogLevel.INFO,
  format: LogFormat.JSON,
};

const defaultTtl = 60;
const momento = new SimpleCacheClient({
  configuration: Configurations.Laptop.latest(loggerOptions),
  credentialProvider: credentialsProvider,
  defaultTtlSeconds: defaultTtl,
});
```

Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: This makes me question how up to date are any of the code examples on this page. @cprice404 any thoughts on how best to have the team audit this?

Comment (allenheltondev) [reply docs/develop/api-reference/index.mdx]: Revised implementation. The example from the SDK docs was not right.

```javascript
const { SimpleCacheClient, EnvMomentoTokenProvider, LogLevel, LogFormat, Configurations } = require('@gomomento/sdk');
const credentials = new StringMomentoTokenProvider ({ authToken: process.env.MOMENTO_AUTH_TOKEN});

const momento = new SimpleCacheClient({
  configuration: Configurations.Laptop.latest(),
  credentialProvider: credentials,
  defaultTtlSeconds: 60
});
```

Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: OK, added that into the file.

File: docs/develop/api-reference/index.mdx [hunk 3]
```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
...
+
+Attributes:
+
+| Name      | Type   | Description                      |
+| --------- | ------ | -------------------------------- |
+| cacheName | String | Name of the cache to be flushed. |
+
+:::tip
+
+While you could use [Delete Cache](#delete-cache), then [Create Cache](#create-cache) to mimic this, the FlushCache API keeps the settings and only deletes the data in the cache.
```
Comment (cprice404) [inline docs/develop/api-reference/index.mdx:211]: is this worth mentioning?

File: docs/develop/api-reference/index.mdx [hunk 4]
```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
...
+
+:::
+
+<SdkExamples
```
Comment (cprice404) [inline docs/develop/api-reference/index.mdx]: we should figure out how we want to validate that these still work since most of these SDKs aren't 1.0 yet

File: docs/develop/api-reference/index.mdx [hunk 5]
```diff
@@ -196,6 +196,96 @@ puts client.caches.to_a.join(", ")
...
+require 'momento'
+MOMENTO_AUTH_TOKEN = "eyJhbGc.MyTestToken"
+DEFAULT_TTL_SECONDS = 15
+client = Momento::SimpleCacheClient.new(
+  auth_token: MOMENTO_AUTH_TOKEN, default_ttl: DEFAULT_TTL_SECONDS
+)
+response = client.flush_cache('test-cache')
+raise repsonse.error if response.error?
+  `}
+  cli={`momento cache flush-cache --name test-cache`}
```
Comment (cprice404) [inline docs/develop/api-reference/index.mdx]: AFAIK this doesn't exist yet, do we still want to document it?  I presume that is true for some of the stuff above like ruby too?

**Review by cprice404 — COMMENTED** [review]
just a few questions

File: docs/develop/api-reference/index.mdx (hunk 3 above)
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx:211]: IMO yes. Why else did we create the feature unless it was different enough from delete and then create?

File: docs/develop/api-reference/index.mdx (hunk 4 above)
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: I am all ears on how we can pay down this technical debt. Should this stop this PR from moving forward?

File: docs/develop/api-reference/index.mdx (hunk 5 above)
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: Correct it doesn't exist yet for all of these. I can remove it for now if you want me to.

File: docs/develop/api-reference/index.mdx (hunk 4 above)
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: Removed the code examples for now.

File: docs/develop/api-reference/index.mdx (hunk 5 above)
Comment (NoSQLKnowHow) [reply docs/develop/api-reference/index.mdx]: Removed the code examples for now.

**Review by allenheltondev — APPROVED** [review]
